packages/patternfly-4/react-core/src/components/Progress/Progress.js

```diff
@@ -21,6 +21,8 @@ const propTypes = {
   variant: PropTypes.oneOf(Object.values(ProgressVariant)),
   /** Title above progress. */
   title: PropTypes.string,
+  /** Label to indicate what progress is shwoing. */
+  label: PropTypes.oneOfType([PropTypes.string, PropTypes.node, PropTypes.number]),
```

Comment:
node includes string and number
```suggestion
  label: PropTypes.node,
```